src/cluster_legacy.c

```diff
@@ -7369,6 +7466,16 @@ const char **clusterCommandExtendedHelp(void) {
         "LINKS",
         "    Return information about all network links between this node and its peers.",
         "    Output format is an array where each array element is a map containing attributes of a link",
+        "IMPORT(-PREPARE) SLOTSRANGE <start slot> <end slot> [<start slot> <end slot> ...]",
```

Comment:
I suggest we seperate IMPORT and IMPORT-PREPARE as 2 commands, because CLUSTER IMPORT is is one shot command, and CLUSTER IMPORT-PREPARE is the first step of  2-phase execution.